src/gui/ApplicationSettingsWidget.cpp

```diff
@@ -308,6 +308,13 @@ void ApplicationSettingsWidget::loadSettings()
                                                       && config()->get(Config::Security_LockDatabaseMinimize).toBool());
     m_secUi->lockDatabaseOnScreenLockCheckBox->setChecked(
         config()->get(Config::Security_LockDatabaseScreenLock).toBool());
+#if defined(Q_OS_MACOS)
+    m_secUi->lockDatabasesOnUserSwitchCheckBox->setEnabled(true);
+#else
+    m_secUi->lockDatabasesOnUserSwitchCheckBox->setEnabled(false);
+#endif
```

Comment:
```suggestion
#if defined(Q_OS_MACOS)
    m_secUi->lockDatabasesOnUserSwitchCheckBox->setVisible(true);
#else
    m_secUi->lockDatabasesOnUserSwitchCheckBox->setVisible(false);
#endif
```
No need to add pointless clutter on other OS. Maybe move this to a different function if `setVisible()` is not appropriate in `loadSettings()`.

Reply:
As I said in the PR text, I want to make sure if the same feature is possible to make with other operating systems. So this visibility will probably change anyway. If it's just for macOS, then the whole option will be invisible, obviously.